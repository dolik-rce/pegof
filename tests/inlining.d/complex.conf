input inlining.d/complex.peg
optimize inline
