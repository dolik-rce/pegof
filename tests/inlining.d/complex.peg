main <- A / B / C

A <- <[Aa]> { printf("A: %s\n", $1); }
B <- <[Bb]> $1
C <- <[Cc]> $1 { printf("C: %s\n", $1); }
