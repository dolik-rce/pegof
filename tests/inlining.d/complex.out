main <-
    (<[Aa]> { printf("A: %s\n", $1); })
    / (<[Bb]> $2)
    / (<[Cc]> $3 { printf("C: %s\n", $3); })
